depends/packages/native_cctools.mk

```diff
@@ -73,6 +73,7 @@ define $(package)_stage_cmds
   cp lib/libLTO.so $($(package)_staging_prefix_dir)/lib/ && \
   cp -rf lib/clang/$($(package)_clang_version)/include/* $($(package)_staging_prefix_dir)/lib/clang/$($(package)_clang_version)/include/ && \
   cp bin/llvm-dsymutil $($(package)_staging_prefix_dir)/bin/$(host)-dsymutil && \
+  cp bin/llvm-config $($(package)_staging_prefix_dir)/bin/ && \
   if `test -d include/c++/`; then cp -rf include/c++/ $($(package)_staging_prefix_dir)/include/; fi && \
   if `test -d lib/c++/`; then cp -rf lib/c++/ $($(package)_staging_prefix_dir)/lib/; fi
 endef
```

Comment:
This package seems to have a different version than Bitcoin, is it OK?

Reply:
Yes, our clang and cctools is the older version. From clang version 8.0.0 the SDK package is downloaded from Xcode package. With this the compilation of simple packages like lib event failed. I guess the package is organised differently. 
 With SDK version 11 boost library does not compile. there are many deprecated errors. So we have to try to upgrade our clang version after changing boost  modules to std.